src/AutoRest.CSharp/Mgmt/Output/ResourceContainer.cs

```diff
@@ -19,6 +19,9 @@ internal class ResourceContainer : ResourceOperation
         private const string ResourceGroupOperationsResourceType = "ResourceGroupOperations.ResourceType";
         private const string SubscriptionOperationsResourceType = "SubscriptionOperations.ResourceType";
         private const string TenantResourceType = "ResourceIdentifier.RootResourceIdentifier.ResourceType";
+        private const string ResourceGroup = "ResourceGroup";
```

Comment:
Can we add CommentName to the end of these const names to make the intention clear for all 3
```suggestion
        private const string ResourceGroupCommentName = "ResourceGroup";
```